keanu-python/.yapfignore

```diff
@@ -1,3 +1,8 @@
 keanu/vertex/generated.py
 keanu/vertex/__init__.py
 build/**
+
+# Need to duplicate patterns for Windows as yapf doesn't seem to do anything special
```

Comment:
It's annoying that we have to do this. I have raised an [issue](https://github.com/google/yapf/issues/661) on the `yapf` repo.

Reply:
Nice one totally agree it should normalise the paths or something.